### ISSUE TITLE
how to train

### ISSUE THREAD

**Issue by bob831009** [arnoweng/CheXNet#4]
Hi, I'm new to pytorch.
I have no idea how to train by this code, because there is only for eval.
Hope you can release the training code. It may be pretty helpful, thx :)

**Comment by zongyuange** [comment]
I think the weighting strategy how the loss weight is assigned to training is pretty important to understand your code. We look forward to your training code.

**Comment by arnoweng** [comment]
Sorry, for some reasons the code is not available now, but you can refer [another implementation](https://github.com/zoogzog/chexnet) for more details.
